From aef4493a4b4b82b3697b166121a0e7a66e1d8f0f Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Thu, 28 May 2015 21:52:53 +0800
Subject: [PATCH 1134/1594] MLK-10992 ARM: dts: imx6: update the model string
 of imx6qp

commit 1e005b8a86fdb3a46a48033744617e356c958a7d from
git://git.freescale.com/imx/linux-2.6-imx.git

For i.MX6QP, the value of '/sys/devices/soc0/machine' should be
"Freescale i.MX6 Quad Plus SABRE Automotive Board". Update the
model string for i.MX6QP.

Signed-off-by: Bai Ping <b51503@freescale.com>
---
 arch/arm/boot/dts/imx6qp-sabreauto.dts |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qp-sabreauto.dts b/arch/arm/boot/dts/imx6qp-sabreauto.dts
index 85260e6..bd6262f 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto.dts
@@ -9,6 +9,10 @@
 #include "imx6q-sabreauto.dts"
 #include "imx6qp.dtsi"
 
+/ {
+	model = "Freescale i.MX6 Quad Plus SABRE Automotive Board";
+};
+
 &mxcfb1 {
 	prefetch;
 };
-- 
1.7.5.4

